### ISSUE TITLE
End of animation

### ISSUE THREAD

**Issue by Brian393** [Brian393/ecotopia-today#2]
@edwardsharp - Today while packing bags for Portland I realized that the animation does not end as I wanted. The idea was:
At the end, or with any user click, the animation geojson is replaced by another geojson containing the full files.
The reason is that the animation should be restricted to only some of the features, and conversely, the full display should include pictures and text, unlike the animation. So it needs two separate files.
If that can be finessed it would be great. 

My other question is: could the same type of animation flash up a small picture and a date? That might be more impressive.

By the way, I think my master is now at least one commit ahead of yours.... It's the commit on my branch Cascadia-5.

**Comment by edwardsharp** [comment]
sure, the non-animating layers are setup via the computed property `watershedDamsTransformationLayers ` https://github.com/Brian393/ecotopia-today/blob/master/src/components/MapWatershed.vue#L110 so just a matter of setting up that layer with whatever .geojson files you'd like. 

re: your other question; sure, that's essentially what the animation is doing now, displaying a small (icon) image and a date. what do you want to do differently?

**Comment by Brian393** [comment]
@edwardhsharp Thanks for answeing. OK, sounds perfect, I will dig into it
on Sunday and see if I can make it work, should be easy as you describe it.
Right now in the airport waiting for Portland flight.

**Comment by Brian393** [comment]
@edwardsharp 

OK, it works great with images! I made some adjustments to the topography and it all looks good.

Now I would like to use the file "watershedDamsTransformation2.geojson" as the final file that appears on click. It includes all the images that  appear in the animation, plus more. I could just replace the current files (Aluminum.geojson, ArmyCorps2.geojson, etc) but they use simple icons, whereas watershedDamsTransformation2.geojson uses a specific image for each icon. So I don't know how to get that going. It will be useful for other things, so I need to learn that.

ALSO, DAMN, I forgot how to get gh-pages in sync with my master. The remote master is up to date and so is the branch Cascadia-7. I tried a buncha things but no success...

I am off traveling for two days so I will see your answer on Thursday.
